persistence/assetlist.json

```diff
@@ -382,6 +382,132 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/stkdydx.svg"
         }
       ]
+    },
+    {
+      "description": "PSTAKE Liquid-Staked STARS",
+      "denom_units": [
+        {
+          "denom": "stk/ustars",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "stkstars",
+          "exponent": 6,
+          "aliases": [
+            "stk/stars"
+          ]
+        }
+      ],
+      "base": "stk/ustars",
+      "name": "PSTAKE staked STARS",
+      "display": "stkstars",
+      "symbol": "stkSTARS",
```

Comment:
Please include the trace to STARS

Reply:
hey @JeremyParish69 trace to stkSTARS wont be applicable right? since it's a native asset on persistence chain. Have added trace for STARS

Reply:
Added traces to all stkAssets as well - https://github.com/cosmos/chain-registry/pull/4029/commits/c020aa7508ac7f63f9f7e6a785be893028d5a678